src/Microsoft.Health.Fhir.Core/Features/Definition/BundleWrappers/BundleEntryWrapper.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Microsoft.Health.Fhir.Core.Features.Definition.BundleWrappers
 {
-    internal class BundleEntryWrapper
+    public class BundleEntryWrapper
```

Comment:
>BundleEntryWrapper [](start = 17, length = 18)

Are change here and BundleWrapper still need considering we switch to `IList<ITypedElement>` away from `BundleWrapper`